src/Math/Direction2.cs

```diff
@@ -245,6 +245,12 @@ public override int GetHashCode()
 
         #endregion
 
+        #region tostring
+
+        public override string ToString() => $"{Radians} rad";
```

Comment:
So you'd rather have people do `direction.Radians.ToString()` if they wanna be specific? I guess that is no more code than otherwise, though you then lose the unit, but you can just do $"{direction.Radians} rad" yourself. I feel that Degrees might be a better default than Radians though, since most people will probably have an easier time with that? Or at least the factor of PI thing? Radians are just.. hard to understand. 6.28...